arch/sim/src/sim/win/sim_hostuart.c

```diff
@@ -36,6 +42,8 @@ static HANDLE g_stdout_handle;
  * Public Functions
  ****************************************************************************/
 
+bool host_uart_checkin(int fd);
```

Comment:
move the simple check function to the beginning?

Reply:
Done